### ISSUE TITLE
Generating a service and automatically providing it, still happens with --dry-run

### ISSUE THREAD

**Issue by gioragutt** [angular/angular-cli#6170]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.

@angular/cli: 1.0.0
node: 7.4.1-nightly20170104e07b6516d7
os: win32 x64
@angular/common: 4.1.0
@angular/compiler: 4.1.0
@angular/core: 4.1.0
@angular/forms: 4.1.0
@angular/http: 4.1.0
@angular/platform-browser: 4.1.0
@angular/platform-browser-dynamic: 4.1.0
@angular/router: 4.1.0
@angular/cli: 1.0.0
@angular/compiler-cli: 4.1.0

### Repro steps.

* `ng g module my-module`
* import `ModuleWithProviders`
* add `static forRoot(): ModuleWithProviders { ... }` to the module
* cd my-module
* ng g service my-service --dry-run --spec=false -m=my-module.module

### The log given by the failure.

There's no error log, but the outcome is that the service (whose file is not even created) is imported in the module file and is added to the `providers` list in the `NgModule`.

### Desired functionality.

`--dry-run` would show that the module file is updated, without updating the module.

**Comment by cexbrayat** [comment]
This has been fixed in 1.0.1 (see https://github.com/angular/angular-cli/commit/be7c9b1b8a1950d92437baf1b36d8a951e2e87dc)

Update your CLI version and it should work.

**Comment by sumitarora** [comment]
@gioragutt Please reopen if it's still an issue.
